graylog2-server/src/test/java/org/graylog2/indexer/indices/IndicesIT.java

```diff
@@ -132,6 +148,18 @@ public void testClose() {
         //assertThat(getIndexState(index)).isEqualTo("close");
     }
 
+    @Test
+    public void testClosedIndices() {
+        final String index1 = client().createRandomIndex("indices_it_");
+        client().closeIndex(index1);
+        final String index2 = client().createRandomIndex("otherindices_it_");
+        client().closeIndex(index2);
+
+        final Set<String> closedIndices = indices.getClosedIndices(Collections.singleton("*"));
+
+        assertThat(closedIndices).containsExactlyInAnyOrder(index1, index2);
+    }
+
     @Test
     public void testAliasExists() {
```

Comment:
Actually I think it would be worth to rename all the tests in here whose names start with `test` to something more meaningful.

Reply:
:heavy_check_mark: